src/Controller/SiteController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+class SiteController extends Controller
+{
+    protected function getId(): string
```

Comment:
I think we need to remove it in the future versions

Reply:
done.